src/Shared/@components/Input/Input.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import s from './Input.module.scss';
+
+export const Input = props => {
```

Comment:
Overall, it's not bad to keep sources of variables explicit so you always know where they come from at the first glance (i.e. using `prop.something` everywhere tells you whether the value was passed in props or computed as soon as you see it, which is cool), but this make this otherwise clean code look cluttered. In the case of reusable components, where we don't really get a lot of injections (something coming in from outside, like props) there's not a lot of value in explicitness, but a lot of it in clarity.

So destructure props. You can even do it like so:

```suggestion
export const Input = ({ name, label, handle change }) => {
```
But you shouldn't and I'll come to why is that in a sec...

Reply:
How about ({ name, label, className, ...props }), senpai?